modules/app/src/main/resources/assets/js/main.ts

```diff
@@ -601,6 +601,10 @@ function initProjectContext(application: Application): Q.Promise<void> {
             return Q(null);
         }
 
+        if (projects.length === 0) {
+            ProjectContext.get().setNoProjects();
```

Comment:
This one should be renamed to `setNotAvailable` as well

Reply:
Sorry, idea didn't show this invocation